packages/language-server/src/completions/debug-twig/types.ts

```diff
@@ -20,10 +20,15 @@ export type TemplatePathMapping = {
     namespace: TemplateNamespace;
 };
 
-export type TwigDebugInfo = {
+export type TwigEnvironment = {
   Filters: TwigFunctionLike[];
   Functions: TwigFunctionLike[];
   Globals: TwigVariable[];
   LoaderPaths: TemplatePathMapping[],
   Tests: string[];
 };
+
+// i'll just leave this alias here
+// to simplify renaming in future
+// TODO: remove
```

Comment:
Will do this on master to reduce the mental overhead of reading this PR's code. [1]